### DISCUSSION TITLE
How can I duplicate current tab

### DISCUSSION THREAD

**Discussion by duoml** [wezterm/wezterm#1770] [Q&A] [answered]
ask for help,thanks

**Comment by wez** [comment, marked as answer]
You can open a new tab using CTRL-SHIFT-T
wezterm will try to preserve the working directory from the current tab.
It's not possible to copy the contents and processes/panes from one tab to another tab.

**Reply by abitrolly** [reply]
@wez how does `wezterm` detect the working directory? I ask, because this feature is broken for me on the latest `wezterm 20240203-110809-5046fc22` release on Linux.
